### ISSUE TITLE
Improvements

### ISSUE THREAD

**Issue by adelefaell** [Minty-Flow/minty-flow#76]
- search input: Clear button is slow and the hit point is narrow.
- when creating a new account sorting should be correct new accounts should be on top
- add new categories presets 
- find a solution for that border that shows onPress hold 
- exclud from balance should also hide it from Total balance in the accounts page
- in preference > privacy: toggling on lock app should be confirmed with a finger scan and vice versa 
- swipe delete is not smoothly opening try making it long or find its issue 
- update all confirm modals to use only titles on alert success and make it simple and clear and only in error to use description or if necessary
